test/integration/kubernetes_deploy_test.rb

```diff
@@ -1,27 +1,118 @@
 require 'test_helper'
 
 class KubernetesDeployTest < KubernetesDeploy::IntegrationTest
-  def test_basic
-    runner = KubernetesDeploy::Runner.new(
-      namespace: @namespace,
-      current_sha: "abcabcabc",
-      context: MINIKUBE_CONTEXT,
-      template_dir: File.expand_path("./test/fixtures/basic"),
-      wait_for_completion: true,
-    )
-    runner.run
-
-    pods = kubeclient.get_pods(namespace: @namespace)
-    managed_pods, unmanaged_pods = pods.partition do |pod|
-      pod.metadata.ownerReferences && pod.metadata.ownerReferences.first.kind == "ReplicaSet"
+  def test_full_basic_set_deploy_succeeds
+    deploy_fixture_set("basic")
+    basic = FixtureSetAssertions::Basic.new(@namespace)
```

Comment:
What's the value of wrapping assertions into `FixtureSetAssertions::Basic.new` if we can use `assert_all_up(@namespace)` as a module method, and include this assertions module into `KubernetesDeploy::IntegrationTest`?